VsViewer/vs_calc/examples/calc_weighted_vs30.py

```diff
@@ -9,15 +9,17 @@
 examples_dir = Path(__file__).parent.resolve()
 cpt_ffps = [examples_dir / "CPT_6457.csv", examples_dir / "CPT_6458.csv"]
 cpts = [CPT.from_file(str(cpt_ffp)) for cpt_ffp in cpt_ffps]
-correlations = ["andrus_2007", "mcgann_2018"]
-correlation_weights = {"andrus_2007": 0.6, "mcgann_2018": 0.4}
-cpt_weights = {"CPT_6457": 0.8, "CPT_6458": 0.2}
+cpt_correlations = ["andrus_2007", "mcgann_2018"]
+cpt_correlation_weights = {"andrus_2007": 0.6, "mcgann_2018": 0.4}
+vs_weights = {"CPT_6457": 0.8, "CPT_6458": 0.2}
+vs30_correlation = "boore_2011"
```

Comment:
I assume this example is just a benchmark for us. However, we've found Boore 2004 is better than Boore 2011 for NZ conditions (in sedimentary basins), so if it is something that is changeable (without consequence), it would probably make more sense to use Boore 2004 so anyone looking at this code would default to the right one (and see that we use the right one in our example). Thoughts?

Reply:
Yea this is just a simple example for people to understand how to use this package to generate a weighted vs30 value. We can use Boore 2004 for sure, these weightings were all just random. Not sure what the actual "proper" method is, but if we find one then we should use that for the default here for sure